### PR TITLE
Add changelog for 5.0.34 release. Add LTS word to all lts releases

### PR DESCRIPTION
**Pull request by UnderGreen**

### CODE REVIEW

File: docs/5.x/changelog.md [hunk 1]
```diff
@@ -125,12 +125,20 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 5.x Releases
 
+### 5.0.34 LTS
+
+### Bugfixes
+* Fix security issue with Kapacitor service.
```
Comment (r0mant) [inline docs/5.x/changelog.md]: If there was a security issue, should these notes include a warning similar to the security releases below? cc @knisbet (and maybe give more info about what the issue was?)

File: docs/5.x/changelog.md [hunk 2]
```diff
@@ -125,12 +125,20 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
...
+
+### Improvements
+* Improved validation of gravity-site upgrade.
```
Comment (r0mant) [inline docs/5.x/changelog.md]: Based on the changelog b/w .33 and .34, how about:
```
Add `postUpdate` hook to `gravity-site` to make sure it's up and running after an upgrade.
```

File: docs/5.x/changelog.md (hunk 1 above)
Comment (knisbet) [reply docs/5.x/changelog.md]: I agree with Roman

Comment (UnderGreen) [reply docs/5.x/changelog.md]: Can you take a look at the new sentence?

Comment (knisbet) [reply docs/5.x/changelog.md]: looks better to me.

**Pull request merged**